examples/cohort_banking_with_sdk/examples/cohort_banking_with_sdk.rs

```diff
@@ -205,39 +198,6 @@ async fn main() -> Result<(), String> {
     Ok(())
 }
 
-fn start_queue_monitor(queue: Arc<Receiver<TransferRequest>>) -> JoinHandle<Result<(), String>> {
```

Comment:
Why we are removing this?

Reply:
It is not really needed. This was useful when replicator was part of `cohort-sdk`.  I forgot to remove it in the earlier PR which was merged to master last week.

Reply:
This one is observing how quickly cohort is draining the generated queue. It is not related to replicator. I prefer it to stay :) What you saw previously is that monitor was printing stats of generator queue and replicator queue (via heartbeat channel), but now it observes only generator. Btw if we remove it, I am not sure how system will stop itself, because async services are running forever, the don't know when to stop.

Reply:
aaah okay, I must have got confused with the earlier logic. Will add this back. 

The app stops correctly even after I remove it. In fact one reason why I removed it was, sometimes, the monitor would preemptively stop the app even before it reached the target number of `txn`. Probably there is a bug somewhere there, which I didn't dig too much in detail, but just deleted the code :)